### DISCUSSION TITLE
Synthesim doesn't support fractional reparmeterization

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#128] [perspectives; sense-making]
I changed every delay time parameters into fractional time, perhaps but due to its tight range (0-1) synthesim doesn't support sliders. So to verify model behavior 
> large `adj mat push frac2` (1/wip adj time) and small `adj mat push frac1` (1/inven adj time) creates oscillation

I need to create extra time parameters to revive slider function. Would there be any way to experiment with sliders without defining extra variables?

![image](https://user-images.githubusercontent.com/30194633/208176075-d54a9329-66df-4542-9812-409b6ebaa02d.png)

**Comment by tomfid** [comment]
Set the Min/Max fields in the equation editor.

Alternately, click on the slider track and set the range there, and check the "make permanent" box.

By clicking on the slider track you can also type in any value, even if it's out of range.
